src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,435 @@
+---
+title: |
+  Boldly link where no one has linked before: Text Fragments
+subhead: |
+  Text Fragments let you specify a text snippet in the URL fragment.
+  When navigating to a URL with such a text fragment, the browser can emphasize
+  and/or bring it to the user's attention.
+authors:
+  - thomassteiner
+date: 2020-04-03
+hero: hero.jpg
+alt:
+description: |
+  Text Fragments let you specify a text snippet in the URL fragment.
+  When navigating to a URL with such a text fragment, the browser can emphasize
+  and/or bring it to the user's attention.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - text-fragments
+  - capabilities
+  - fugu
+draft: true
+---
+## Fragment Identifiers
+
+Chrome&nbsp;80 was a big release.
+It contained a number of highly anticipated features like
+[ECMAScript Modules in Web Workers](https://web.dev/module-workers/),
+[nullish coalescing](https://v8.dev/features/nullish-coalescing),
+[optional chaining](https://v8.dev/features/optional-chaining), and more.
+The release was, as usual, announced through a
+[blog post](https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html)
+on the Chromium blog.
+You can see an excerpt of the blog post in the screenshot below.
+
+<figure class="w-figure">
+  <img src="blog-red-ids.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Chromium blog post with red boxes around elements with an <code>id</code> attribute.</figcaption>
+</figure>
+
+You are probably asking yourself what all the red boxes mean.
+They are the result of running the following snippet in DevTools.
+It highlights all elements that have an `id` attribute.
+
+```js
+document.querySelectorAll('[id]').forEach((el) => {
+  el.style.border = 'solid 2px red';
+});
+```
+
+I can place a deep link to any element highlighted with a red box thanks to the
+[fragment identifier](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web#Fragment)
+which I then use in the [hash](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash)
+of the page's URL.
+Assuming I wanted to deep link to the *Give us feedback in our
+[Product Forums](http://support.google.com/bin/static.py?hl=en&page=portal_groups.cs)*
+box in the aside, I could do so by handcrafting the URL
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<mark class="highlight-line highlight-line-active">#HTML1</mark></code></a>.
+As you can see in the Elements panel of the Developer Tools, the element in question
+has an `id` attribute with the value `HTML1`.
+
+<figure class="w-figure">
+  <img src="id-html1.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Dev Tools showing the <code>id</code> of an element.</figcaption>
+</figure>
```

Comment:
This image is rather small on account of `width="400"`

![image](https://user-images.githubusercontent.com/4713486/81128962-9bca1580-8ef7-11ea-8d43-98111abad6c3.png)

Reply:
Now 600 :-)